tool/tsh/tsh.go

```diff
@@ -703,7 +703,7 @@ func Run(ctx context.Context, args []string, opts ...cliOption) error {
 	play.Arg("session-id", "ID of the session to play").Required().StringVar(&cf.SessionID)
 
 	// scp
-	scp := app.Command("scp", "Secure file copy")
+	scp := app.Command("scp", "Transfer files over SFTP")
```

Comment:
nit:
Can you remove the SFTP part? Now I think it's confusing as SCP uses SFTP. I think that the fact that the SCP command copies files should be enough. The underlying protocol is an implementation detail.

Reply:
Done